hid-uclogic-rdescs.h

```diff
@@ -0,0 +1,48 @@
+#ifndef UCLOGIC_REPORT_DESCRIPTORS
+#define UCLOGIC_REPORT_DESCRIPTORS
+
+#include <linux/device.h>
+
+/* Size of the original descriptor of WPXXXXU tablets */
+#define UCLOGIC_WPXXXXU_RDESC_ORIG_SIZE	212
+
+/* Size of the original descriptor of the new WP5540U tablet */
+#define UCLOGIC_WP5540U_V2_RDESC_ORIG_SIZE	232
+
+/* Size of the original descriptor of WP1062 tablet */
+#define UCLOGIC_WP1062_RDESC_ORIG_SIZE	254
+
+/* Size of the original descriptor of PF1209 tablet */
+#define UC_LOGIC_PF1209_RDESC_ORIG_SIZE	234
```

Comment:
Could you please write `UCLOGIC` here, without the underscore, to match all the other names?